R/load-init.R

```diff
@@ -20,6 +20,11 @@
 #' load_init(dir = dir, init = "INIT_VMPA_Jan2015.nc", vars = c("Planktiv_S_Fish2_ResN",
 #'                                                              "Planktiv_S_Fish3_ResN"))
 #' load_init(dir = dir, init = "INIT_VMPA_Jan2015.nc", vars = "Megazoobenthos_N")
+#'
+#' dir <- "C:/Users/siebo/Documents/Atlantis/BalticAtlantis/run_files_73days_Nej"
+#' init <- "new_init_Baltic_05Dec2015_v2.nc"
+#' vars <- "Sprat1_ResN"
+#' load_init(dir = dir, init = init, vars = vars)
 
```

Comment:
Great work. I moved the expand_limits call in custom_wrap into plot_line. Therefore, the optional yexpand parameter is not needed in custom_wrap anymore. The error on TravisCI was due to a missing update of the documentation. Thanks alot!